src/query-builder/SelectQueryBuilder.ts

```diff
@@ -4322,11 +4322,18 @@ export class SelectQueryBuilder<Entity extends ObjectLiteral>
 
                     if (InstanceChecker.isFindOperator(where[key])) {
                         if (
-                            where[key].type === "moreThan" ||
-                            where[key].type === "lessThan"
+                            [
+                                "moreThan",
+                                "lessThan",
+                                "moreThanOrEqual",
+                                "lessThanOrEqual",
+                            ].includes(where[key].type)
                         ) {
                             const sqlOperator =
-                                where[key].type === "moreThan" ? ">" : "<"
+                                (where[key].type.startsWith("moreThan")
+                                    ? ">"
+                                    : "<") +
+                                (where[key].type.endsWith("OrEqual") ? "=" : "")
```

Comment:
code works, however I would prefer a simpler code. simple if-elseif statements work better for the code understanding.

Reply:
I took this to mean that you wanted an if-block just for `moreThanOrEqual` and `lessThanOrEqual`, even if meant repeating largely the same block of code. Let me know if this is not what you meant!

Reply:
yeah I meant that... but not exactly such way....

Reply:
I thought you would just change it to:

```ts
 where[key].type === "moreThan" ||
where[key].type === "lessThan" ||
where[key].type === "lessThanOrEqual" ||
where[key].type === "lessThanOrEqual"
```

and then just another check... okay just revert it, previous was better.

Reply:
sorry for the confusion.